maps_dashboards/public/components/map_container/map_container.scss

```diff
@@ -8,7 +8,7 @@
 /* stylelint-disable no-empty-source */
 .map-container {
   width: 100%;
-  min-height: calc(100vh - 98px);
+  min-height: calc(100vh - 154px);
```

Comment:
Why are we hardcoding these offsets?

Reply:
Offline talked with Ashwin, will update here like: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_builder/public/application/app.scss#L13